eStore/Core/UI/Color.swift

```diff
@@ -0,0 +1,34 @@
+//
+//  Color.swift
+//  eStore
+//
+//  Created by Vladislav Kondrashkov on 4/7/19.
+//  Copyright © 2019 Vladislav Kondrashkov. All rights reserved.
+//
+
+import UIKit
+
+struct Color {
+    // Navigation bar
```

Comment:
I think these comments are kind of misleading as they can quickly become outdated and inconsistent. I would suggest removing them at all 🙂

Reply:
Sure, as I wanted, it was kind of temporary comments to find out what’s color is used for